programming-extensions/programming-extension-pnpssl/src/main/java/org/objectweb/proactive/extensions/ssl/PASslEngine.java

```diff
@@ -25,12 +25,12 @@
  */
```

Comment:
I'm currently considering to upgrade the bouncycastle dependencies to version 1.77  (in the root _build.gradle_):
- org.bouncycastle:bcprov-jdk15to18:1.77
- org.bouncycastle:bcprov-ext-jdk15to18:1.77


I suggest to compile and test you changes using the new version to avoid issues or conflicts.

Reply:
I doubt the new version would remove these algorithms, but I will test it.

Reply:
@nebilbenmabrouk I upgrade bc locally and the pnps tests pass without issues.